tests/protobuf_to_fd.test/runit

```diff
@@ -0,0 +1,41 @@
+#### Verify that serializing protobuf directly to fd reduces memory footprint
+
+#!/usr/bin/env bash
+bash -n "$0" | exit 1
+
+dbnm=$1
+
+set -e
+
+# Make sure we talk to the same host
+host=`cdb2sql --tabs ${CDB2_OPTIONS} $dbnm default 'SELECT comdb2_host()'`
+
+cdb2sql $dbnm --host $host 'CREATE TABLE t (b BLOB)'
+
+sleep 1
+
+# Insert a 32MB blob and read it back.
+cdb2sql $dbnm --host $host 'INSERT INTO t VALUES (randomblob(33554432))'
```

Comment:
Don't think we need to do any of this. Simpler test to eliminate memory used by berkeleydb, bdb cursor etc: `select randomblob(33554432)`